conda_forge_tick/auto_tick.xsh

```diff
@@ -66,8 +66,19 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
                                    fork=fork,
                                    gh=gh)
 
-    # migrate the `meta.yaml`
     recipe_dir = os.path.join(feedstock_dir, 'recipe')
+    # if postscript/activate no noarch
+    script_names = ['pre-unlink', 'post-link', 'pre-link', 'activate']
+    exts = ['.bat', '.sh']
+    no_noarch_files = [
+        '{}.{}'.format(script_name, ext)
+        for script_name in script_names for ext in exts
+        ]
+    if migrator.__class__.__name__ == 'Noarch' and any(
```

Comment:
I think this should be `if isinstance(migrator, Noarch)`, rather than checking the classname